api/libproject/devstart.rb

```diff
@@ -205,36 +211,29 @@ def do_run_migrations(creds_file, project)
   common = Common.new
   create_db_file = Tempfile.new("#{project}-create-db.sql")
   begin
-    unless system("cat db/create_db.sql | envsubst > #{create_db_file.path}")
-      raise("Error generating create_db file; exiting.")
-    end
+    common.run_inline("cat db/create_db.sql | envsubst > #{create_db_file.path}")
```

Comment:
My experience was that commands like this (where we're actually executing multiple commands in the shell) don't work with run_inline. That's why I was using system in places.

Reply:
OK, I reverted the ones with pipes/redirects to use system() and pulled out a helper for that case to reduce unless..raise duplication, echo the commands as they're run, and have a single place to comment on why system() is required.

I also elided some of the temp files, since I think they can just be piped also -- any reason not to do that?